packages/internal/dex/sdk-sample-app/src/config/devnet.ts

```diff
@@ -15,8 +15,8 @@ const immutableConfig = new ImmutableConfiguration({
 const contractOverrides: ExchangeContracts = {
   multicall: '0x9482D1727424B6C3EeaA22B037FFBC3ae6748f66',
   coreFactory: '0x8081d5F526b7Aaf4868e6C53Aa8a9d9D93c10562',
-  quoterV2: '0xC12B5c73951CFD922979638b5d19C593ac51dcDA',
-  peripheryRouter: '0x8089b5D6fa3f19C64081d5050c5CA3a66f34C5af',
+  quoter: '0xC12B5c73951CFD922979638b5d19C593ac51dcDA',
+  swapRouter: '0x8089b5D6fa3f19C64081d5050c5CA3a66f34C5af',
   secondaryFee: '0x8089b5D6fa3f19C64081d5050c5CA3a66f34C5af', // not deployed currently
```

Comment:
Should we rename this one to swapProxy? I actually thought i already did

Also why does it say "not deployed currently" ?

Reply:
Deployed here using ol faithful https://blockscout.dev.immutable.com/address/0x0234ceca85Efb0c3a751088d328F3db3d397DDBF/contracts#address-tabs

Reply:
Also this PR has your changes but wasn't merged, so doing that now https://github.com/immutable/ts-immutable-sdk/pull/1315